packages/features/eventtypes/components/EventType.tsx

```diff
@@ -62,10 +62,11 @@ export const EventType = (
     isPlatform?: boolean;
     tabName: (typeof tabs)[number];
     tabsNavigation: VerticalTabItemProps[];
+    allowDelete?: boolean;
   }
 ) => {
   const { formMethods, isPlatform, tabName } = props;
-  const { eventType, team, currentUserMembership, tabMap, isUpdating } = props;
+  const { eventType, team, currentUserMembership, tabMap, isUpdating, allowDelete = true } = props;
```

Comment:
I think it's more clear if default true value is set in props type
```
allowDelete?: boolean = true;
```

Reply:
agreed